### PR TITLE
Bookstore

### PR DESCRIPTION
**Pull request by Tracy-miranja**
Set up a new React application using [Create-react-App].

Create a directory for your reusable components: /components.

Installed [React Router]

Set up your router.

Add 2 routes in my application:

which contains:

- [ ] A book list, where each book has a title and author
- [ ]     A form that allows for creating a new book
- [ ]     /categories
Include a navigation component that includes links to the 2 routes you created.

never used style for my project.

### CODE REVIEW

**Review by arslanbisharat — COMMENTED** [review]
Hi @Tracy-miranja, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have correct readme file as required my Microverse. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Tracy-miranja** [comment]
Hi @arslanbisharat ,i don't get you when you say invalid readme, can you please elaborate

**Comment by arslanbisharat** [comment]
Dear,

You are not using correct Readme Template.

Looking forward to hearing from you.


*Best,*Arslan Bisharat

**Comment by Tracy-miranja** [comment]
Thanks @arslanbisharat i saw that i have corrected, thanks for the feedback

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @Tracy-miranja , 

Good job so far. !
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights

- Professional readme file
- No linters error
- Good PR description

## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [ ] Please as part of the requirements, you are supposed to have a form that allows for creating a new book just below the books list in your book component. ill recommend creating a component for the form and import it in your `Book` component.
image



_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Categories.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+function Categories() {
+  return (
+    <>
+
```
Comment (danmainah) [inline src/components/Categories.js:4]: - [ ] [optional] @Tracy-miranja great work implementing the changes requested, however, the last reviewer forgot to point out that in the categories page you need a button that checks for status (it doesn't work now but acts as a placeholder for future features implementation)

**Review by danmainah — APPROVED** [review]
Hi @Tracy-miranja , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Categories.js (hunk 1 above)
Comment (Tracy-miranja) [reply src/components/Categories.js:4]: Hi @danmainah thanks for the feedback, ill check on that.

**Pull request merged**